modules/http-helper/dummy_server.go

```diff
@@ -46,6 +46,39 @@ func RunDummyServerE(t *testing.T, text string) (net.Listener, int, error) {
 	return listener, port, err
 }
 
+// RunDummyHandlerServer runs a dummy HTTP server on a unique port that will serve given handlers. Returns the Listener for the server,
```

Comment:
Ok just fix these comments then I think we're good to run the tests.

Reply:
Fixed :wink: